client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -30,8 +31,7 @@ import ReaderSubscriptionListItemPlaceholder
  * @param {String} url - the url to format
  * @returns {String} - the formatted url.  e.g. "https://www.wordpress.com/" --> "wordpress.com"
  */
-const formatUrlForDisplay = url =>
-	untrailingslashit( url.replace( /^https?:\/\/(www\.)?/, '' ) );
+const formatUrlForDisplay = url => untrailingslashit( url.replace( /^https?:\/\/(www\.)?/, '' ) );
```

Comment:
looks like that line is 97 characters long.  @jsnajdr figured out that `prettier-atom` ignores the forced command line args so your editor is probably defaulting to a printWidth of 80 instead of the intended 100.

I'll see if i can hardcode it in the fork so that atom will pick up the settings properly